### DISCUSSION TITLE
Password?

### DISCUSSION THREAD

**Discussion by Dutsken** [EternityBots/Nezuko#17] [Q&A]
Usarname for 'https//github.com': 
Password for 'https://name@github.com:

Tenho dificuldades de passar desta etapa. Há algum usarname e password especifico aqui para liberar a execução da clonagem?

**Comment by Ray-Senpai07** [comment]
Não é necessário nenhum nome de usuário e senha para clonar nosso repositório git. Você pode usar uma URL
